scout/parse/variant/genotype.py

```diff
@@ -324,24 +326,23 @@ def get_ref_depth(
     """Get reference read depth"""
     ref_depth = int(variant.gt_ref_depths[pos])
     if ref_depth == -1:
```

Comment:
This is weird, why is this == -1

Reply:
Since "." in the GT call?

Reply:
```suggestion
    if ref_depth != -1:
```

Reply:
Sry, didn't think about the return at first.